File: src/stats/model/Summary.kt
package stats.model

external interface Summary {

    @JsName("data_at")
    var dataAt: String

    var repositories: Array<SummaryRepository>

    @JsName("total_downloads_count")
    var totalDownloadsCount: Int

    @JsName("total_stargazers_count")
    var totalStargazersCount: Int

    @JsName("total_open_issues_count")
    var totalOpenIssuesCount: Int

    @JsName("total_open_issues_url")
    var totalOpenIssuesUrl: String

    @JsName("total_open_pull_requests_count")
    var totalOpenPullRequestsCount: Int

    @JsName("total_open_pull_requests_url")
    var totalOpenPullRequestsUrl: String

    @JsName("total_commits_since_release")
    var totalCommitsSinceRelease: Int

    var url: String
}

File: src/stats/model/Summary.ts
import { SummaryRepository } from './SummaryRepository';

export interface Summary {
  data_at: string;

  repositories: SummaryRepository[];

  total_downloads_count: number;

  total_stargazers_count: number;

  total_open_issues_count: number;

  total_open_issues_url: string;

  total_open_pull_requests_count: number;

  total_open_pull_requests_url: string;

  total_commits_since_release: number;

  url: string;
}
